.evergreen/functions.yml

```diff
@@ -677,7 +677,7 @@ functions:
           npm i -w packages/compass-smoke-tests https://x-access-token:${generated_token}@github.com/10gen/compass-mongodb-com --engine-strict=false
 
           if [[ "$IS_WINDOWS" == "true" ]]; then
-            # TODO: windows_setup
+            npm run --unsafe-perm --workspace @mongodb-js/compass-smoke-tests start -- --package=windows_setup --tests=time-to-first-query
```

Comment:
you probably want auto-update-from? This will be basically the only one that actually upgrades in CI. Assuming it works 😆